apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -104,11 +103,15 @@ const ContributionsTable = ({
           id: contribution.id,
           details: contribution.details,
           proof: contribution.proof,
+          updatedAt: contribution.updatedAt,
           date_of_submission: contribution.date_of_submission,
           engagementDate: formatDate(contribution.date_of_engagement),
+          date_of_engagement: formatDate(contribution.date_of_engagement),
```

Comment:
I know we have some redundancy here. We should aim to refactor these to match -- this is from before we were strict with typing.